primitives/src/constants.rs

```diff
@@ -94,6 +94,7 @@ parameter_types! {
     pub const MaxAssets: u16 = MaxCategories::get() + 1;
     pub const MaxInRatio: Balance = (BASE / 3) + 1;
     pub const MaxOutRatio: Balance = (BASE / 3) + 1;
+    pub const MaxSwapFee: Balance = BASE / 10; // 10%
```

Comment:
This has been taken directly from the Balancer smart contracts: https://github.com/balancer-labs/balancer-core/blob/f4ed5d65362a8d6cec21662fb6eae233b0babc1f/contracts/BConst.sol#L25

DEXs seem to take something between 0% and 1%, 0.3% is a very common number. However, I'd say that the case of prediction markets is different. Money moves even faster than usual, and LPs stand to lose a lot of money, while traders can win big at any given time. So allowing the LPs to charge a hefty trading fee is fine, I think. But anything above 10% fees rediculously high.

The frontend will display the trading fees so that users aren't ambushed with huge trading fees.

... And of course, this number is still up for discussion when we set the on-chain constants.